### PR TITLE
Add placeholder page for the New Contributor Orientation.

### PR DESCRIPTION
**Pull request by jberkus**
Add a placeholder page so we have something to link to.

I dropped it in Documentation because that's where the New Contributor Course is.   It could also be in Events or Resources.

@kaslin @mfahlandt please review!

### CODE REVIEW

File: content/en/docs/orientation/_index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+If you're new to the Kubernetes community and looking for a place to contribute,
+it can be challenging to figure out how to get started.  New contributors should
+join us for one of the New Contributor Orientation sessions, held once a month
+on Zoom.
+
+**Meeting Time**: every 3rd Tuesday of the month for 1 hour
+
+<a href="https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NXVpdGhoMWRyMGhpMDZjd[…]afef04s12ra0gkqql6fchjc%40group.calendar.google.com&scp=ALL" >EMEA/APAC-friendly: 1:30 PT / 8:30 UTC / 10:30 CET / 14:00 IST</a>
+
+<a href="https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MnZqMXVmazZhNWJ2aTNld[…]afef04s12ra0gkqql6fchjc%40group.calendar.google.com&scp=ALL" >AMER-friendly: 8:30 PT / 15:30 UTC / 17:30 CET / 21:00 IST</a>
```
Comment (sftim) [inline content/en/docs/orientation/_index.md:18]: These hyperlinks look truncated. Otherwise, LGTM.

Comment (jberkus) [reply content/en/docs/orientation/_index.md:18]: Feh, @mfahlandt ?  Those are your links.

**Comment by kaslin** [comment]
Oh we can also add the github page: https://github.com/kubernetes/community/tree/master/mentoring/new-contributor-orientation
and the playlist now: https://youtu.be/gWQb6OfG2CE

**Comment by jberkus** [comment]
Updated to remove most of the info and direct to the GH page instead.  Keeping this page because it's a better URL and search than GH pages.

@kaslin, please approve.

File: content/en/docs/orientation/_index.md (hunk 1 above)
Comment (jberkus) [reply content/en/docs/orientation/_index.md]: Solved this by removing the links entirely.

**Comment by kaslin** [comment]
I think it's perfect! Thank you, @jberkus!
/lgtm
/approve

**Pull request merged**